src/components/labelStandalone/CdrLabelStandalone.vue

```diff
@@ -8,7 +8,6 @@
         {{ label }}{{ required || optional ? '' : '' }}
         <span
           v-if="required"
-          aria-label="required"
```

Comment:
`aria-label="required"` is not needed here. Since the input element already has `aria-required=true` all this does is make screen readers say "Required" twice.